main.py

```diff
@@ -327,6 +327,37 @@ def index(marker=None, message=None):
             context[attr] = value or '-1'
     if message:
         context['message'] = message
+    pref_accident_severity = []
+    pref_light = ('prefLight', '2', 'קלה'.decode("utf8"))
+    pref_severe = ('prefSevere', '1', 'חמורה'.decode("utf8"))
```

Comment:
I think it will be clearer to define a class here so you can refer to these fields by name in the template rather than by index (which is bound to change and break). Then here you'll have constructor calls.